app/components/works/edit/form_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: 'works/form_fields', locals: { work_form: @work_form, content: @files } %>
```

Comment:
I found this partial rendering approach was required mainly due to the turbo stream component.

Reply:
Wrapping all of this in a component doesn't seem necessary. Can't you just pass in the content to the slot instead of using a component?

Reply:
I'll see. I ran into some issues earlier, but I may have worked those out and not come back to this.

Reply:
Done